### PR TITLE
Update photo collection screen

### PR DESCRIPTION
**Pull request by Finemas**
Add pacient name in title of navigation bar and remove option to add photo from photo gallery expect the simulator.

### CODE REVIEW

File: zScanner/ViewControllers/NewDocument/NewDocumentPhotosViewController.swift [hunk 1]
```diff
@@ -52,7 +52,11 @@ class NewDocumentPhotosViewController: BaseViewController {
     let bottomGradientOverlayHeight: CGFloat = 80
     
     @objc private func takeNewPicture() {
-        showActionSheet()
+        #if targetEnvironment(simulator)
```
Comment (skorepak) [inline zScanner/ViewControllers/NewDocument/NewDocumentPhotosViewController.swift]: Better to use the condition `UIImagePickerController.isSourceTypeAvailable(.camera)` 
Sorry If I told you to make condition directly for simulator, We just need to be able to add images on simulator and this condition should do it.

File: zScanner/ViewModels/NewDocument/NewDocumentPhotosViewModel.swift [hunk 2]
```diff
@@ -14,9 +14,11 @@ class NewDocumentPhotosViewModel {
     
     // MARK: Instance part
     private let tracker: Tracker
+    let folderName: String
     
-    init(tracker: Tracker) {
+    init(tracker: Tracker, folderName: String) {
         self.tracker = tracker
+        self.folderName = folderName
```
Comment (skorepak) [inline zScanner/ViewModels/NewDocument/NewDocumentPhotosViewModel.swift:21]: good job 👏

**Review by skorepak — CHANGES_REQUESTED** [review]
Minor comment added

File: zScanner/ViewControllers/NewDocument/NewDocumentPhotosViewController.swift (hunk 1 above)
Comment (Finemas) [reply zScanner/ViewControllers/NewDocument/NewDocumentPhotosViewController.swift]: No problem. I can do it but it already works. We are able to add images from gallery on simulator.

Comment (skorepak) [reply zScanner/ViewControllers/NewDocument/NewDocumentPhotosViewController.swift]: It’s better to use the swift code then compiler flags. 

And it covers edge cases better.

**Review by skorepak — APPROVED** [review]

**Pull request merged**